docs/data/toolpad/connecting-to-data-sources/function.md

```diff
@@ -0,0 +1,55 @@
+# Function
+
+<p class="description">
+    Function query type is an advanced way to make request from Toolpad app.
+</p>
+
+## Working with functions
+
+As explained in the [connections](/toolpad/connecting-to-data-sources/connections/) section you can either create a reusable `connection` or simply create a new `query` and put all connection details inline
+
+1. Choose `ADD QUERY` in the `Properties editor` on the right
+
+1. Select `Function` type and click `CREATE QUERY`
+
+   ![Function type](/static/toolpad/function-query-1.png)
+
+1. We got all the basic configuration settings as described in [queries](/toolpad/connecting-to-data-sources/queries/) section
+
+1. In addition we are now presented with a `code editor` where we can write a custom code for data fetching
+
+   ![Function configuration](/static/toolpad/function-query-2.png)
+
+   Supported features:
+
+   - Subset of webplatform APIs
+
+     - fetch (Request, Response)
+     - AbortController
+     - console (.log, .debug, .info, .warn, .error)
+     - setTimeout, clearTimeout
+     - TextEncoder, TextDecoder
+     - ReadableStream
+
+   - Access outside variables by binding `Parameters` fields
+   - `Console` and `Network` tabs for an easier debugging
+
+   Current limitations:
+
+   - We can not import modules
+
+1. Once finished with configuration click `SAVE` and click anywhere outside of the dialog to return to the editor
+
+## Usecases
```

Comment:
Is it usecase, use case or use-case? In the core repo I see almost everywhere "use case", twice "use-case", and never "usecase"

```suggestion
## Use cases
```